source/api_documentation/signal_api/_create_call_outcome.rst

```diff
@@ -75,8 +75,8 @@
         "advertiser_campaign_id_from_network": "2",
         "network_id": "3"
       },
-      "call_outcome": {
-        "outcome_name": "sale",
+      "signal": {
+        "name": "sale",
         "partner_unique_id": "1",
```

Comment:
The csv template shows `signal_name`, but all the other attributes are not prefixed.  I believe the file upload will also work with `name` but it made sense to me to prefix this one attribute for clarity sake.